src/sql/workbench/contrib/notebook/browser/notebookEditor.ts

```diff
@@ -192,7 +192,7 @@ export class NotebookEditor extends BaseEditor implements IFindNotebookControlle
 
 		const parentElement = this.getContainer();
 
-		super.setInput(input, options, CancellationToken.None);
+		await super.setInput(input, options, CancellationToken.None);
```

Comment:
found this missing await while reviewing code. The call to setInput is quite fast so in practice this is likely not causing any issues.